streamdl_proto_srv.py

```
@@ -104,43 +142,73 @@ def GetStream(self, request, context):
+                    context.set_details(f"{res['error']} - Internal server error")
                 case _:
                     context.set_code(grpc.StatusCode.UNKNOWN)
                     context.set_details(f"{res['error']} - Unknown error")
             return pb.StreamResponse(error=error_code)
 
 
 def serve():
+    # Start HTTP health server
+    health_port = 8080
+    # Bind to all interfaces for container networking (safe in isolated container environment)
+    health_server = HTTPServer(("0.0.0.0", health_port), HealthHandler)  # nosec B104
```

Ruff (0.15.9)

[error] Possible binding to all interfaces

(S104)